scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/keystone-api.sh.j2

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -eux
+set -o pipefail
+
+export VENV_FOLDER="{{ ardana_qe_test_venv }}"
+
+set +u
+source $VENV_FOLDER/bin/activate
+set -u
+
+source /var/lib/ardana/keystone.osrc
+NOAUTH_KSADMINTOKEN=`openstack token issue | grep '| id' | awk '{print $4}'`
```

Comment:
Why using `grep`, `awk`? you can achieve the same using only the openstack-client:

`openstack token issue -f value -c id`